### DISCUSSION TITLE
setclientcharacter console command does not need cheat to use

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14289] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

you do not need to enablecheats to use setclientcharacter in multiplayer

### Reproduction steps

1. start a multiplayer campaign
2. use setclientcharacter to control outpost npc

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
you can also set a invalid character to achieve freecam

**Comment by Regalis11** [comment]
This is actually intentional: the command may have some valid "non-cheat" uses, such as letting players take over bots.
